cobra/flux_analysis/sampling.py

```diff
@@ -14,27 +14,47 @@
 
 import numpy as np
 import pandas
-from cobra.solvers import get_solver_name, solver_dict
-from cobra.util import create_stoichiometric_matrix, nullspace
+from sympy.core.singleton import S
+from cobra.util import (create_stoichiometric_matrix, constraint_matrices,
+                        nullspace, assert_optimal)
 
-BTOL = np.finfo(np.float32).eps
+bounds_tol = np.finfo(np.float32).eps
 """The tolerance used for checking bounds feasibility."""
 
-FTOL = BTOL
+feasibility_tol = bounds_tol
```

Comment:
My apologies for being unclear, with lowercase vars I meant local variables not the globals which are fine as all caps.